.travis.yml

```diff
@@ -2,18 +2,11 @@ sudo: required
 dist: trusty
 language: scala
 scala:
-  - 2.11.7
+  - 2.11.8
 jdk:
   - oraclejdk8
-before_install:
-  - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 1397BC53640DB551
-  - sudo apt-get update -qq
-  - sudo apt-get install -qq mysql-server-5.6 mysql-client-5.6 mysql-client-core-5.6
-  - docker pull ubuntu:latest
-before_script:
-  - mysql -u root -e "SET GLOBAL sql_mode = 'STRICT_ALL_TABLES';" 
-  - mysql -u root -e "CREATE DATABASE IF NOT EXISTS cromwell_test;"
-  - mysql -u root -e "CREATE USER 'travis'@'localhost' IDENTIFIED BY '';"
-  - mysql -u root -e "GRANT ALL PRIVILEGES ON cromwell_test . * TO 'travis'@'localhost';"
-script: sbt -Dbackend.providers.Local.config.filesystems.local.localization.0=copy clean coverage nointegration:test coverageReport && sbt coverageAggregate && sbt assembly
-after_success: sbt coveralls
+env:
+  - BUILD_TYPE=test
+  - BUILD_TYPE=centaur
```

Comment:
Indeed.  Perhaps add a comment as to the transcendental nature of what's happening here.